recipes/python-kaleido/LICENSE.txt

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
```

Comment:
Looks like this file is in the tarball, so you don't need to include it separately.

Reply:
Good point!

Reply:
Missed removing this file.

Reply:
Right, thanks! 01f22f1